Description: Custom defaults for debian package
Author: Léo Haf <leo@haf.ovh>
Forwarded: not-needed
Last-Update: 2026-04-10
--- a/share/examples/bitcoin.conf
+++ b/share/examples/bitcoin.conf
@@ -52,7 +52,7 @@
 
 # Specify directory to hold blocks subdirectory for *.dat files (default:
 # <datadir>)
-#blocksdir=<dir>
+datadir=/var/lib/bitcoin/
 
 # Whether to reject transactions from network peers. Disables automatic
 # broadcast and rebroadcast of transactions, unless the source peer
@@ -843,7 +843,7 @@
 #rpcwhitelistdefault=1
 
 # Accept command line and JSON-RPC commands
-#server=1
+server=1
 
 
 ### Statistic options
