Description: Custom defaults for debian package
Author: Léo Haf <leo@haf.ovh>
Forwarded: not-needed
Last-Update: 2026-04-10
--- a/share/examples/bitcoin.conf
+++ b/share/examples/bitcoin.conf
@@ -89,7 +89,7 @@
 #daemonwait=1
 
 # Specify data directory
-#datadir=<dir>
+datadir=/var/lib/bitcoin/
 
 # Maximum database cache size <n> MiB (minimum 4, default is platform
 # dependent, between 100 and 2048). Make sure you have enough RAM.
@@ -863,7 +863,7 @@
 #rpcwhitelistdefault=1
 
 # Accept command line and JSON-RPC commands
-#server=1
+server=1
 
 
 ### Statistic options
