Description: Desktop file linux with desktop environments
Author: Léo Haf <leo@haf.ovh>
Forwarded: not-needed
Last-Update: 2025-07-22

--- /dev/null
+++ b/contrib/debian/bitcoin-knots-gui.desktop
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Bitcoin Knots
+Comment=a graphical interface for interacting with Bitcoin Knots
+Exec=/usr/bin/bitcoin-qt
+Icon=/usr/share/bitcoin/bitcoin1024.png
+Terminal=false
+Type=Application
+Categories=Finance;Network;
+StartupWMClass=BitcoinKnots
+Keywords=node
+SingleMainWindow=true
\ No newline at end of file
